test/chainrules.jl

```diff
@@ -37,6 +37,12 @@
                     test_scalar(airyaiprimex, x)
                 end
             end
+
+            # finite differencing fails for x = 0
```

Comment:
This is wrong, I misinterpreted the ChainRules tests. I always mix up what "expected" and "actual" refer to.

Reply:
I added the limit of the derivative at 0 as a special case.